### PR TITLE
Bug 1913585: fix text for `Create a Project`

### PR DESCRIPTION
**Pull request by sahil143**
fixes: https://issues.redhat.com/browse/ODC-5309

fix text from `Create a Project` => `create a Project.`

Screenshot 2021-01-07 at 1 37 17 PM

### CODE REVIEW

**Comment by jerolimov** [comment]
@sahil143 Please change also the keys in the other translation files (ja and zh locals)

**Comment by jerolimov** [comment]
After discussion in team sync we just change the english version, so everything is fine here. :smile: :+1:

/lgtm

**Review by jerolimov — APPROVED** [review]

File: frontend/packages/dev-console/src/components/projects/CreateProjectListPage.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const CreateProjectListPage: React.FC<CreateProjectListPageProps> = ({
     <ProjectListPage {...props} title={title}>
       {children} {t('devconsole~or')}{' '}
       <Button isInline variant="link" onClick={openProjectModal}>
-        {t('devconsole~Create a Project')}
+        {t('devconsole~create a Project.')}
       </Button>
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/components/projects/CreateProjectListPage.tsx:34]: This string should be translated as a single translation and not a concatenation. You can use the `<Trans` component.

Comment (sahil143) [reply frontend/packages/dev-console/src/components/projects/CreateProjectListPage.tsx]: fixed

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**